src/expressions/closure-expr.md

```diff
@@ -1,5 +1,11 @@
 # Closure expressions
 
+> **<sup>Syntax</sup>**  
+> _ClosureExpression_ :  
+> &nbsp;&nbsp; `move`<sup>?</sup>  
+> &nbsp;&nbsp; ( `||` | `|` [_FunctionParameters_]<sup>?</sup> `|` )  
+> &nbsp;&nbsp; ([_Expression_] | `->` [_TypeNoBounds_]&nbsp;[_BlockExpression_])  
```

Comment:
Is `TypeNoBounds` actually defined anywhere?

Reply:
There is a draft: https://brauliobz.github.io/rust-reference/types.html#types .